commands/authenticate.go

```diff
@@ -22,10 +22,17 @@ type AuthenticateConn interface {
 // Authenticate is an AUTHENTICATE command, as defined in RFC 3501 section
 // 6.2.2.
 type Authenticate struct {
-	Mechanism string
+	Mechanism       string
+	InitialResponse string
```

Comment:
Can we change this to be a `[]byte`, and encode to base64 in `Command`?

Reply:
Sure, 2766c70.